tests/test_integration_org_unit.py

```diff
@@ -22,13 +22,14 @@ class Tests(util.LoRATestCase):
 
     def test_org_unit_temporality(self):
         self.load_sample_structures()
-
```

Comment:
Erm, newlines are good — they make the code more readable. Any particular reason to kill this one?

Reply:
I have no opinions on this and will put it back. I do not know why it was removed, most likely old habit because I am used to another convention for this.